extensions/wikia/PotentialMemberPageExperiments/scripts/entry-point-experiment.js

```diff
@@ -57,15 +71,35 @@ require([
 			action: tracker.ACTIONS.CLICK
 		});
 		$banner.remove();
-		setDismissCookie();
+		onBannerDismissed();
+	}
+
+	function isEntryPointInViewport() {
+		return bannerOffset > w.scrollY &&
+			bannerBottomOffset < (w.scrollY + w.innerHeight);
+	}
+
+	function trackEntryPointInViewport() {
+		if (isEntryPointInViewport()) {
+			trackBannerImpression();
+			$(w).off('scroll.trackPMPEntryPoint');
+		}
+	}
+
+	function trackBannerImpression() {
+		track({
+			label: '',
```

Comment:
I'm working on tracking viewability of the banner. It's just a placeholder and if I don't use it in the end I'll remove it in the upcoming PR. What do you think? :)